plugin/grpctrace/interceptor.go

```diff
@@ -426,7 +426,7 @@ func peerInfoFromContext(ctx context.Context) []core.KeyValue {
 	return peerInfoFromTarget(p.Addr.String())
 }
 
-var fullMethodRegexp = regexp.MustCompile(`^/\S*\.(\S*)/\S*$`)
+var fullMethodRegexp = regexp.MustCompile(`^/(?:\S*\.)?(\S*)/\S*$`)
 
 func serviceFromFullMethod(method string) string {
 	match := fullMethodRegexp.FindAllStringSubmatch(method, 1)
```

Comment:
The following `if` statement will still cause a panic if this new regex is not matched. A check should be put in place to prevent this general type of panic.

Reply:
Got it, thanks for the review! I'll address it tomorrow (europe time here), after #684 adds the base testing.
Also, see my [comment](https://github.com/open-telemetry/opentelemetry-go/issues/682#issuecomment-621996367) on #682 . There might be something I'm doing wrong.

Reply:
changed

Reply:
> Changed. I am a bit unsure about the beginning '/'. The [gRPC](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#appendix-a---grpc-for-protobuf) reference does not mention that service names start with a '/':
> `Service-Name → ?( {proto package name} "." ) {service name}`.
> Maybe this can also be optional? So we don't have surprises if they decide to change the naming schema in the future.

+1

Agreed that doesn't seem correct based on the reference.